examples/python/src/main/resources/python/take_sqrt.py

```diff
@@ -0,0 +1,7 @@
+import numpy as np
+
+
+def transform_list(input_list):
```

Comment:
I would add some python style docs here describing the input parameters etc as standard in python